src/scss/components/_tabs.scss

```diff
@@ -1,8 +1,22 @@
+$tabs-focused-outline: none !default;
+$tabs-link-focus-active-border-bottom-color: $tabs-link-active-border-bottom-color !default;
+$tabs-link-focus-border-bottom-color: $tabs-link-hover-border-bottom-color !default;
+
 .b-tabs {
     .tabs {
         margin-bottom: 0;
         flex-shrink: 0;
         li {
+            a:focus {
+                outline: $tabs-focused-outline;
+                border-bottom-color: $tabs-link-focus-active-border-bottom-color;
+            }
+            &:not(.is-active) {
+                a:focus {
+                    border-bottom-color: $tabs-link-focus-border-bottom-color;
+                }
+            }
+
```

Comment:
this is good for the the default type but other types of tab should have their `:focus` match to the `:hover` state